concepts/05 Widgets/Form/01 Getting Started with Form/80 Submit the Form.md

```diff
@@ -0,0 +1,264 @@
+To submit the Form, wrap it in the HTML <a href="https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form" target="_blank">form</a> element. Then add the [button](/Documentation/ApiReference/UI_Widgets/dxForm/Item_Types/ButtonItem/) item and set its **buttonOptions**.[useSubmitBehavior](/Documentation/ApiReference/UI_Widgets/dxButton/Configuration/#useSubmitBehavior) property to **true**. You can submit the Form data to the server only when all validation checks pass successfully.  
```

Comment:
> You can submit the Form data to the server

Can we change it
Note: Form will submit data to the server only when all validation checks pass successfully.

Reply:
Implemented this one.